joomla/jissues
# Update g11n lib - and others..

Pull Request for Issue #1043 .

#### Summary of Changes
* Update the g11n library
* Remove clue/graph-composer because of conflicts with symfony/console
* Since I was unable to only update the g11n lib, several other packages have also been upgraded.

#### Testing Instructions
Please check that everything works as expected, including CLI scripts...

`@TODO` Remove the PHP 7.0 check from Travis and move the 7.2 check out of "Allowed failures"

Fix #1043
